### PR TITLE
Registration closed functionality

### PR DESCRIPTION
**Pull request by lavanya-bmw**

### CODE REVIEW

File: src/styles/newApp.css [hunk 1]
```diff
@@ -631,3 +631,14 @@ textarea {
...
 }
+
+.registration-close-content {
+  flex: 1;
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 50px 50px;
+  border-radius: 25px;
+  text-align: center;
+  background: #f8f8f8 !important;
```
Comment (jjeroch) [inline src/styles/newApp.css:642]: @lavanya-bmw this is without any background transparency, right? Can we please change this to "white"

**Review by jjeroch — CHANGES_REQUESTED** [review]

**Review by nidhigarg-bmw — APPROVED** [review]
Looks good to me

**Review by oyo — APPROVED** [review]
looks good to me :)

**Review by jjeroch — APPROVED** [review]

**Comment by jjeroch** [comment]
@lavanya-bmw can you recheck the merge conflicts?

**Comment by jjeroch** [comment]
@lavanya-bmw I merged the change. Hope thats fine

**Pull request merged**